server/project/accounts/models.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# Create your models here.
+
+from django.db import models
+from django.utils import timezone
+
+ACCOUNT_TYPES = (('a', 'Asset'), ('l', 'Liability'), ('e', 'Equity'))
+BOOL_TYPES = (('t', 'True'), ('f', 'False'), ('n', 'Null'))
+
+
+class Account(models.Model):
+    account_id = models.IntegerField(primary_key=True)
+    account_name = models.CharField(max_length=60, unique=True)
+    priority = models.IntegerField(verbose_name="Priority (0 is top)", default=0)
+    is_active = models.CharField(max_length=1, choices=BOOL_TYPES, default='t')
```

Comment:
Not sure why this field is nullable. With that said, why not use a BooleanField for this value? https://docs.djangoproject.com/en/1.11/ref/models/fields/#booleanfield

Reply:
Addressed